spring-web/src/main/java/org/springframework/http/client/reactive/ReactorNetty2ResourceFactory.java

```diff
@@ -107,8 +107,9 @@ public void addGlobalResourcesConsumer(Consumer<HttpResources> consumer) {
 	 * Use this when you don't want to participate in global resources and
 	 * you want to customize the creation of the managed {@code ConnectionProvider}.
 	 * <p>By default, {@code ConnectionProvider.elastic("http")} is used.
-	 * <p>Note that this option is ignored if {@code userGlobalResources=false} or
-	 * {@link #setConnectionProvider(ConnectionProvider)} is set.
+	 * <p>Note that this supplier is ignored if {@link #isUseGlobalResources()}
```

Comment:
@izeye Is it expected that `false` is changed to `true` here is addition to the code -> link refinement? cc @simonbasle